app.py

```
@@ -1,100 +1,121 @@
-    raw   = chain.invoke(
-        {"system": SYSTEM, "question": q.question, "context": ctx}
-    )
-    sql_code = sqlparse.format(raw, strip_comments=True).strip()
-    return {"sql": sql_code, "ddl_context": ctx}
+    prompt= TEMPLATE.format(system=SYSTEM_TEXT, question=q.question, context=ctx)
+    raw   = llm.invoke(prompt)
+
+    sql_plain = strip_md_fence(raw)
+    sql_fmt   = sqlparse.format(sql_plain, strip_comments=True).strip()
+
+    first_token = re.sub(r"^[\s:>\-]+", "", sql_fmt.lstrip()).split()[0].lower()
```

Ruff (0.11.9)

Local variable `first_token` is assigned to but never used

Remove assignment to unused variable `first_token`

(F841)